sdk/identity/identity/review/identity.api.md

```diff
@@ -74,6 +74,21 @@ export class AzureCliCredential implements TokenCredential {
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
 }
 
+// @public
+export class AzurePowerShellCredential implements TokenCredential {
+    constructor(options?: AzurePowerShellCredentialOptions);
+    protected getAzurePowerShellAccessToken(resource: string): Promise<{
```

Comment:
Do we worry about putting these in the API and making them difficult to change later?

Reply:
This shouldn't be here. My bad!